### ISSUE TITLE
`bootc-image-builder` is missing from the developer guides

### ISSUE THREAD

**Issue by supakeen** [osbuild/osbuild.github.io#81]
[bootc-image-builder](https://github.com/osbuild/bootc-image-builder) is missing from the project(s) pages in the developer guides.

**Comment by ochosi** [comment]
Indeed. We only pull in the readme at this point and that goes to https://osbuild.org/docs/bootc/ because we want to advertise it as a separate product.

Would you suggest to duplicate the readme?
